app/build.gradle

```diff
@@ -26,9 +26,9 @@ android {
 dependencies {
     implementation project(path: ':rxandroidbleserver')
 
-    implementation 'io.reactivex.rxjava2:rxjava:2.2.17'
-    implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
-    implementation 'com.github.tbruyelle:rxpermissions:0.10.2'
+    implementation 'io.reactivex.rxjava3:rxjava:3.0.0'
+    implementation 'io.reactivex.rxjava3:rxandroid:3.0.0'
+    implementation 'com.github.niklasvd96:rxpermissions:master-SNAPSHOT'
```

Comment:
Avoid using snapshots to allow reproducible builds. Please create a release and use the version number instead.

Reply:
fixed